sim-administration/simmanager/src/main/kotlin/org/ostelco/simcards/admin/PeriodicProvisioningTask.kt

```diff
@@ -14,11 +14,7 @@ import org.ostelco.prime.simmanager.NotFoundError
 import org.ostelco.prime.simmanager.SimManagerError
 import org.ostelco.simcards.hss.HssEntry
 import org.ostelco.simcards.hss.SimManagerToHssDispatcherAdapter
-import org.ostelco.simcards.inventory.HssState
-import org.ostelco.simcards.inventory.SimEntry
-import org.ostelco.simcards.inventory.SimInventoryDAO
-import org.ostelco.simcards.inventory.SimProfileKeyStatistics
-import org.slf4j.LoggerFactory
+import org.ostelco.simcards.inventory.*
```

Comment:
Explicit imports are better (I think)

Reply:
Fixed, also updated my intellij preferences to don't do it again.